spec/support/platform_helpers.rb

```diff
@@ -65,6 +65,12 @@ def windows_gte_10?
   Gem::Requirement.new(">= 10").satisfied_by?(Gem::Version.new(win32_os_version))
 end
 
+def windows_11?
+  return false unless windows?
+
+  Gem::Requirement.new(">= 10.0.22621").satisfied_by?(Gem::Version.new(win32_os_version))
```

Comment:
How have we come up with version number `10.0.22621` for windows11?

Reply:
That version comes from the OS itself if you grab the version from inside Windows or you can use powershell and run this:

`Get-CimInstance Win32_OperatingSystem`

The version above is the build number returned by that command